src/coprocessor/dag/executor/mod.rs

```diff
@@ -136,6 +137,9 @@ pub trait Executor {
     fn next(&mut self) -> Result<Option<Row>>;
     fn collect_output_counts(&mut self, counts: &mut Vec<i64>);
     fn collect_metrics_into(&mut self, metrics: &mut ExecutorMetrics);
+    fn take_eval_warnings(&mut self) -> Vec<select::Error> {
```

Comment:
Shall we remove the default implementation to avoid forgetting to implement it?

Reply:
rest LGTM

Reply:
Maybe add some comments would be better, just like the following `stop_scan` did? We give the default implementation since only executors with eval computation need to implement `take_eval_warnings`.